x/nft/username/handler.go

```diff
@@ -14,21 +16,27 @@ const (
 func RegisterRoutes(r weave.Registry, auth x.Authenticator, issuer weave.Address) {
 	bucket := NewBucket()
 	r.Handle(pathIssueTokenMsg, NewIssueHandler(auth, issuer, bucket))
+	r.Handle(pathAddAddressMsg, NewAddChainAddressHandler(auth, issuer, bucket))
+	r.Handle(pathRemoveAddressMsg, NewRemoveChainAddressHandler(auth, issuer, bucket))
 }
 
 // RegisterQuery will register this bucket as "/nft/usernames"
 func RegisterQuery(qr weave.QueryRouter) {
 	NewBucket().Register("nft/usernames", qr)
 }
 
-type IssueHandler struct {
+type tokenHandler struct {
```

Comment:
this type might be of use on a parent nft level, but I don't insist on it now.
You'll have to make it public then and also we might have other use-cases where it might be the case.

Reply:
👍 let's refactor as needed.